CODE_STYLE.md

```diff
@@ -28,6 +28,11 @@ For multiline strings, we prefer to use `textwrap.dedent`:
 
 (from https://docs.python.org/3/library/textwrap.html#textwrap.dedent)
 
+## Errors
+
+Error messages must say what happened, why it happened and what you can do to
+fix it.
```

Comment:
We're changing how we actually test the errors as well, right? That would probably be worth noting here.

Reply:
Good point, added.